packages/react-router/tests/link.test.tsx

```diff
@@ -1,44 +1,805 @@
 import React from 'react'
-import { describe, it, expect, afterEach } from 'vitest'
-import { render, cleanup } from '@testing-library/react'
+import '@testing-library/jest-dom/vitest'
+import { afterEach, describe, expect, it, test, vi } from 'vitest'
+import { cleanup, fireEvent, render, screen } from '@testing-library/react'
 
 import {
-  createRouter,
+  createLink,
+  createMemoryHistory,
   createRootRoute,
   createRoute,
-  createMemoryHistory,
-  RouterProvider,
-  createLink,
+  createRouter,
   Link,
+  RouterProvider,
+  useLoaderData,
+  useSearch,
+  redirect,
+  useRouteContext,
+  useParams,
+  Outlet,
 } from '../src'
 
 afterEach(() => {
+  window.history.replaceState(null, 'root', '/')
   cleanup()
 })
 
 describe('Link', () => {
-  it('should NOT pass the "disabled" prop to the rendered Link component', async () => {
+  test('when a Link is disabled', async () => {
```

Comment:
Is there any reason `test` is being used instead of `it`?

Reply:
I personally prefer actually no nesting in tests. And to juse use `test`. I like my tests super simple